BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.1.0-preview2-02629-01
+2.1.0-preview3-02631-01
```

Comment:
We'll get the rc1 version when I fix up the updater. (Technically the preview3 one was built on BuildTools master, but there was recently a mop-up merge from master to release/2.1 in BuildTools so this isn't a problem.)